app/templates/_proposition_header.html

```diff
@@ -10,7 +10,10 @@
       <ul id="proposition-links">
         {% if current_user.has_role('admin') or current_user.has_role('admin-ccs-category') %}
         <li>
-            <a href="{{ url_for('main.service_update_audits') }}">Service Updates</a>
+            <a href="{{ url_for('main.service_update_audits') }}">Service updates</a>
+        </li>
+        <li>
+            <a href="{{ url_for('main.service_status_update_audits') }}">Service status changes</a>
```

Comment:
I'm not sure what qualifies a page as getting it's own top-level navigation rather than another item in the huge list of things on the main admin page?  This is fine here as far as I'm concerned, but we should take a proper look at what goes where in the Admin app some time soon.